transifex/native/tools/migrations/mark.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+"""This module contains everything related to the functionality
+that marks a migration to Transifex Native code for proofreading.
+
+When a file or a string in a file is marked for proofreading, a special
+string is added in the file, so that users can grep for it and
+make sure the migration is correct.
+"""
+
+from __future__ import unicode_literals
+
+from transifex.common.console import Color
+from transifex.native.tools.migrations.models import Confidence
+
+MARK_PROOFREAD_STRING = 'Transifex Native: PROOFREAD_STRING'
+MARK_PROOFREAD_FILE = 'Transifex Native: PROOFREAD_FILE'
+
+
+def mark_string(string_migration, comment_format, mark):
+    """Mark a string migration for proofreading.
+
+    Adds a comment in the beginning of this string.
+
+    :param StringMigration string_migration: the string migration object
+        to update
+    :param unicode comment_format: the format of the comment,
+        compatible with the file type of the migration
+    :param str mark: the string to add as a comment
+    """
+    string_migration.update(
+        '', comment_format.format(mark),
+        append=False,
+    )
+
+
+class MarkPolicy(object):
+    """Determines if a migrated file or a migrated string will be marked
+    for proofread.
+    """
+
```

Comment:
`name` should be defined here as well. Also, you may consider `__NAME` instead of `name`. That should be a protected constant. In that case, you can add the get `name()` function to return that name back.

Reply:
Ideally yes, however I find it extremely unlikely for someone to abuse this because it's public. I think it's OK to leave it as is.

Reply:
@dbendilas I am good with that